third_party/2and3/yaml/__init__.pyi

```diff
@@ -20,12 +20,34 @@ def load_all(stream: Union[str, IO[str]], Loader=...) -> Iterator[Any]: ...
 def safe_load(stream: Union[str, IO[str]]) -> Any: ...
 def safe_load_all(stream: Union[str, IO[str]]) -> Iterator[Any]: ...
 def emit(events, stream=..., Dumper=..., canonical=..., indent=..., width=..., allow_unicode=..., line_break=...): ...
-def serialize_all(nodes, stream=..., Dumper=..., canonical=..., indent=..., width=..., allow_unicode=..., line_break=..., encoding=..., explicit_start=..., explicit_end=..., version=..., tags=...): ...
+
+@overload
+def serialize_all(nodes, stream: None=..., Dumper=..., canonical=..., indent=..., width=..., allow_unicode=..., line_break=..., encoding=..., explicit_start=..., explicit_end=..., version=..., tags=...) -> AnyStr: ...
```

Comment:
A return value of AnyStr doesn't make sense if there's no AnyStr in the arguments.

Reply:
For me `AnyStr` was just a shortcut for `Union[str, unicode]` on Python2 and `Union[str, bytes]` on Python3. I replaced it with my own construct for now.